src/internal_rep.rs

```diff
@@ -248,6 +248,21 @@ impl Declared for TypeInfo {
 
 impl TypeInfo {
     pub fn new(td: TypeDecl, file: &SimpleFile<String, String>) -> Result<TypeInfo, CascadeErrors> {
+        let mut temp_vec = td.inherits.to_vec();
```

Comment:
td.inherits is already a vector.  I think just `clone()` instead of `to_vec()` would be clearer?